playground/src/app/products-filters/state/products-filters.store.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
```

Comment:
We're almost at the end, but we need documentation. Do you think you can create a markdown file that documents how to use the plugin and the API it exposes? Don't worry about the English; I will help with that.

Reply:
Ok, I will make à documentation.

And for the filter store name ?

Reply:
I will do this when you finish everything.